### DISCUSSION TITLE
Structure and Vicuna Usage

### DISCUSSION THREAD

**Discussion by ghost** [ggml-org/llama.cpp#2578] [Polls]
Hi, **<3** llama.cpp

A solution is currently being worked on to resolve `EOS` behavior in `interactive`: [EOSFix](https://github.com/ggerganov/llama.cpp/pull/2689#issuecomment-1686807325)

Related: https://github.com/ggerganov/llama.cpp/issues/2507, https://github.com/ggerganov/llama.cpp/pull/2646#issuecomment-1684988735, https://github.com/ggerganov/llama.cpp/issues/2598, https://github.com/ggerganov/llama.cpp/issues/2417

Thank you.

**Comment by ossirytk** [comment]
I have tested this model today, and it seems to run just fine.I have had no issues with other models either. This might be an issue with your params or prompt.

**Reply by ghost** [reply]
Hi, thanks for your message, but others also experience the issue: wtarreau [bisected and determined](https://github.com/ggerganov/llama.cpp/pull/2646#issuecomment-1684988735)`in-prefix-bos` to be a real regression.

ggerganov [identified the problem](https://github.com/ggerganov/llama.cpp/pull/2646#issuecomment-1685193819): so we'll likely see a fix soon.
